app/models/contact_details.rb

```diff
@@ -1,11 +1,11 @@
 class ContactDetails < ApplicationRecord
   MAX_LENGTHS = {
-    address: 250,
+    phone_number: 35,
     email_address: 250,
-    phone_number: 35
+    address: 250
   }.freeze
 
-  validates :address, :email_address, :phone_number, presence: true
+  validates :phone_number, :email_address, :address, presence: true
```

Comment:
This is long-distance coupling between the view and the model, as the order of fields is defined in the view markup. Solving that problem is possible, but probably more trouble than it's worth at this stage?

Reply:
Yeah, this is a good point. I agree with you that this is more trouble than it's worth at this stage as it's highly likely that the fields are going to change with the application not in sync with the prototype rn.